package.json

```diff
@@ -44,5 +44,10 @@
     "lint",
     "prettier",
     "vscode"
-  ]
+  ],
+  "peerDependenciesMeta": {
```

Comment:
@rtritto could we please move this up so that it is right below the `peerDeps` key?

Many thanks in advance 🙌🏼